t/03-testapi.t

```diff
@@ -472,6 +472,70 @@ subtest 'check_assert_screen' => sub {
     };
 };
 
+subtest 'upload_logs' => sub {
+    $bmwqemu::vars{AUTOINST_URL_HOSTNAME} = 'localhost';
+    $bmwqemu::vars{QEMUPORT} = '4242';
+    $bmwqemu::vars{JOBTOKEN} = 'LookAtMeImAToken';
+    $cmds = [];
+    upload_logs '/var/log/messages';
+    is_deeply($cmds, [
+            {
+                'text' => 'curl --form upload=@/var/log/messages --form upname=basetest-messages http://localhost:4243/LookAtMeImAToken/uploadlog/messages',
```

Comment:
The Perl style check doesn't like the quotes around the keys here:

```
5: ::warning file=/__w/os-autoinst/os-autoinst/t/03-testapi.t,line=483,col=17,title=Hash key 'text' with quotes - severity 4::[HashKeyQuotes] Avoid useless quotes
```

(If it was me we wouldn't enforce this and I think we also don't enforce this in openQA.)

Reply:
I updated it.
In general though, could we either change that or provide a perltidy config that automatically deals with it?
@perlpunk

Reply:
It doesn't seem to be supported? https://metacpan.org/dist/Perl-Tidy/view/bin/perltidy